.github/workflows/stack.yaml

```diff
@@ -0,0 +1,67 @@
+name: "Full-stack tests"
+
+on: push
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [macOS-latest, ubuntu-latest]
+        python-version: [3.8, 3.9]
+
+    env:
+      CI_OS: ${{ matrix.os }}
+      PYVER: ${{ matrix.python-version }}
+      COV: --cov=openff/units --cov-report=xml --cov-config=setup.cfg --cov-append
+
+    steps:
+    - uses: actions/checkout@v2
+
+    # Each of these checkouts can be done on a feature branch
+    # https://github.com/actions/checkout#checkout-a-different-branch
+
+    - name: Checkout OpenFF Toolkit
+      uses: actions/checkout@v2
+      with:
+        repository: openforcefield/openff-toolkit
+        path: openff-toolkit
+
+    - name: Checkout OpenFF Evaluator
+      uses: actions/checkout@v2
+      with:
+        repository: openforcefield/openff-evaluator
+        path: openff-evaluator
+
+    - name: Checkout OpenFF Interchange
+      uses: actions/checkout@v2
+      with:
+        repository: openforcefield/openff-interchange
+        path: openff-interchange
+
+    - name: Look around
+      run: |
+        pwd
+        ls -a
+        ls openff-*/
+
+    - uses: conda-incubator/setup-miniconda@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+        mamba-version: "*"
+        environment-file: devtools/conda-envs/full.yaml
+        auto-activate-base: false
+
+    - name: Install most recent development versions
+      shell: bash -l {0}
+      run: |
+        python -m pip install -e openff-toolkit/
+        python -m pip install -e openff-evaluator/
+        python -m pip install -e openff-interchange/
```

Comment:
Are the pip installs guaranteed to completely supersede the conda-installed packages? I'd be a bit cautious about cases where we've removed/renamed a file since the last release, but the remnants of a conda-installed version of the package makes it seem like it's still around.

Reply:
I don't think that guarantee can be made for the case you describe. The better approach here would be to `conda remove --force openff-x` but some strange stuff has happened locally that spooks me; last week I had both Evaluator and Toolkit removed when I only specified the toolkit ... let's see what happens here.

Reply:
The logs that I checked were not spooky.